quiz/views.py

```diff
@@ -0,0 +1,62 @@
+from django.shortcuts import render, get_object_or_404, HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+
+from quiz import models
+
+
+def index(request):
+
+    quizzes = models.Quiz.objects.all()
+
+    return render(request, 'quiz/index.html', {'quizzes': quizzes})
+
+
+def get_quiz_by_pk(request, pk):
+
+    quiz = get_object_or_404(models.Quiz, pk=pk)
+    questions = quiz.questions.all()
+    return render(request, 'quiz/quiz_detail.html', {'quiz': quiz, 'questions': questions})
+
+
+@csrf_exempt
+def submit_quiz(request, pk):
```

Comment:
As your views code gets bigger and bigger, you may consider transforming it into a callable class to better highlight the functions related to the view as methods.